azure-pipelines.yml

```diff
@@ -717,14 +717,16 @@ jobs:
       BuildDropPath: $(Build.ArtifactStagingDirectory)
 
   - task: PublishPipelineArtifact@0
-    displayName: 'Publish Artifact: yum-mariner'
+    displayName: 'Publish Artifact: rpm-mariner'
     inputs:
       TargetPath: $(Build.ArtifactStagingDirectory)
-      ArtifactName: yum-mariner
+      ArtifactName: rpm-mariner
 
 
-- job: BuildYumPackage
-  displayName: Build Yum Package
+- job: BuildRpmPackageCentOS7
+  displayName: Build Rpm Package CentOS 7
+  # Uncomment this line to disable this job on Pull Requests
+  # condition: and(succeeded(), in(variables['Build.Reason'], 'IndividualCI', 'BatchedCI', 'Manual', 'Schedule'))
```

Comment:
This line was removed by https://github.com/Azure/azure-cli/pull/20608. Bring it back as a comment.